### PR TITLE
feat: unruggable

### PR DESCRIPTION
**Pull request by ponderingdemocritus**
WIP

- implements unruggable meme factory

### CODE REVIEW

File: packages/agent/src/index.ts [hunk 1]
```diff
@@ -21,7 +21,8 @@ import Database from "better-sqlite3";
 import fs from "fs";
 import readline from "readline";
 import yargs from "yargs";
-import { character } from "./character.ts";
```
Comment (fabianhug) [inline packages/agent/src/index.ts:24]: please remove custom `blobert` and revert changes to original `character`

**Review by fabianhug — CHANGES_REQUESTED** [review]
see comment, didnt check the rest yet

**Comment by ponderingdemocritus** [comment]
this is WIP for now

**Comment by fabianhug** [comment]
> this is WIP for now

great! can you mark it as a `Draft PR` until ready? Thanks!

**Pull request merged**